### PR TITLE
Fixing allow downsample online wav nnet3 decoding

### PR DESCRIPTION
**Pull request by bayethiernodiop**

### CODE REVIEW

File: src/feat/online-feature.cc [hunk 1]
```diff
@@ -77,7 +77,7 @@ template<class C>
 void OnlineGenericBaseFeature<C>::AcceptWaveform(BaseFloat sampling_rate,
                                                  const VectorBase<BaseFloat> &waveform) {
   BaseFloat expected_sampling_rate = computer_.GetFrameOptions().samp_freq;
-  if (sampling_rate != expected_sampling_rate)
+  if (sampling_rate != expected_sampling_rate && !computer_.GetFrameOptions().allow_downsample)
```
Comment (danpovey) [inline src/feat/online-feature.cc]: Actually it's more complicated than this because there is an allow_upsample flag too.

Comment (danpovey) [reply src/feat/online-feature.cc]: you don't seem to be on the right path here, you are just ignoring the mismatch but doing nothing.  @jtrmal would you mind making a PR for this?

**Comment by jtrmal** [comment]
I'll have a look.
y.

File: src/feat/online-feature.cc (hunk 1 above)
Comment (bayethiernodiop) [reply src/feat/online-feature.cc]: Ah ok sorry. anyway thx.

**Pull request closed without merging**